source/form.html

```diff
@@ -81,44 +81,51 @@ <h1 class="selection__title">
           </div>
           <fieldset class="filling-form__fieldset filling-form__fieldset--owner-data">
             <legend class="filling-form__legend filling-form__legend--owner-data">
-              Контактные данные (владельца кота)
+              <span>
+                Контактные данные<br>
+                (владельца кота)
+              </span>
```

Comment:
```suggestion
                Контактные данные (владельца кота)
```
span и br совершенно лишние, можно обойтись без них